lib/html-report.js

```diff
@@ -0,0 +1,81 @@
+import readPkgUp from 'read-pkg-up'
+import mkdirp from 'mkdirp'
+import fs from 'fs'
+import path from 'path'
+
+const toSizes = modules => {
+  const sizes = {}
+  for (const { id: idRaw, size } of modules) {
+    const id = idRaw.replace(/^\0(?:commonjs-proxy:)?/, '')
```

Comment:
I'm somewhat torn on including this as is. I think it is logical to remove that for most users. My only hesitation is, since that id is attributable to a plugin and not Rollup core should it be built in? While unlikely, it has slippery slope potential.

Another potential argument is that particular prefix should stay to alert users that the module in question likely isn't benefiting from tree-shaking.

All that to say, everything else looks good so I'm going to merge it then add an option for modifying ids. That will apply to both the text output as well as the chart.

Reply:
Yeah, I'm not crazy about it either. The \0 prefix seems like a pretty standard thing in Rollup land, so that could be separate from the CommonJS plugin prefix. I just hardcoded it because the plugin is so common.